prometheus_rules/alerts/alerts.yaml

```diff
@@ -1,57 +1,46 @@
-groups:
-- name: slo-exporter-slo-error-budget-alerts
-  rules:
-
-  - alert: ErrorBudgetExhausted
-    expr:
-            slo:stable_version{enabled!="false"}
-            * on(slo_version, slo_domain, namespace) group_right()
-            slo:violation_ratio{slo_time_range="4w"}
-            / on (slo_class,slo_domain,slo_version,slo_type,namespace) group_left ()
-            (
-              slo:violation_ratio_threshold - 1
-            )
-            +1 <= 0
-    labels:
-      severity: critical
-    annotations:
-      title: 'Error budget is exhausted.'
-      description: '{{$labels.slo_type | title}} error budget for SLO domain "{{$labels.slo_domain}}" was exhausted.'
-
-
 # SLO burn-rate alerts implementing Multiwindow, Multi-Burn-Rate Alerts (https://landing.google.com/sre/workbook/chapters/alerting-on-slos/)
 #
-#
 # Calculation:
 # 1. It uses pre-calculated burn-rates over two time ranges
-#   1.1 It is multiplied by `slo:stable_version{enabled!="false"}` to select the current stable enabled slo version
-#   1.2 It is compared with a burn-rate threshold selected for the given time range
+#   1.1 It is multiplied by the rate coefficient
+#   1.2 It is multiplied by `slo:stable_version{enabled!="false"}` to select the current stable enabled slo version
+#   1.3 It is compared with a burn-rate threshold selected for the given time range
 # 2. Both conditions are joined with an `and` on specified labels
 #
 # Note:
 # 1. Every alert has a long window and a short window. The long window is the important one.
 #    The shorter window is supposed to prevent alert firing after an issue has been solved.
+# 1. Rate coefficient is used to adjust the burn-rate according to current rate of events.
+#    Both time windows use the coefficient for the rate in the long window.
+
+groups:
 
 # One hour alert
 #
 # The burn-rate threshold of 13.44 is such that given an average request rate,
 # the alert would fire on consuming 2 % of 28-day error budget over the last hour.
 #
-
 - name: slo-burn-rate-1h-alerts-critical
   rules:
   - alert: SloOneHourAlert
     expr: (
         slo:burn_rate{slo_time_range="1h"}
+        * on(slo_domain, slo_class, slo_version, slo_type, namespace) group_left() slo:events_rate_coefficient{slo_time_range="1h"}
```

Comment:
I'm not personally against adding the slo:events_rate_coefficient into the alerts, but we already agreed in one of the previous PRs to keep the example rules as simple as possible.

@FUSAKLA, are you open to revisit this?

Reply:
as we discussed, it will be same as our internal deployment.